.github/actions/setup-environment/action.yml

```diff
@@ -50,9 +50,13 @@ runs:
       shell: bash
       working-directory: "./eden"
     - name: Setup eve version
-      if: "${{ inputs.eve_tag != '' }}"
+      if: ${{ inputs.eve_tag != '' }} and contains(${{ inputs.eve_tag }}, ":" )
```

Comment:
That means we will just ignore any input which does not contain `:` which might be misleading because input variable name is `eve_tag`, I'd put this check inside `run:` section that way this action can use eve_tag which could be just tag or contain registry. Because this way if you want to run eden tests for specific tag inside eden repository you'll have to specify lf-edge/eden:$tag$ as eve tag which IMO redundant. 
Either this or changing name to `eve_image`

Reply:
Will change the inputs to `eve_image` to be more consistent.